src/main/java/com/aws/greengrass/lifecyclemanager/KernelAlternatives.java

```diff
@@ -232,6 +249,7 @@ public static Path locateCurrentKernelUnpackDir() throws IOException, URISyntaxE
         if (unpackDir == null || ! Files.exists(unpackDir) || !Files.isDirectory(unpackDir.resolve(KERNEL_BIN_DIR))) {
             throw new IOException("Unable to locate the unpack directory of Nucleus artifacts");
         }
+#endif /* ANDROID */
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, delete these tokens

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.